src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,59 @@
+'use client';
+
+import clsx from 'clsx';
+import React from 'react';
+
+import './TextArea.css';
+
+interface TextAreaProps {
+  value: string;
+  onChange: (e: React.ChangeEvent<HTMLTextAreaElement>) => void;
+  onSubmit?: () => void;
+  placeholder?: string;
+  width?: number;
+  height?: number;
```

Comment:
```suggestion
  width?: string; // 예: 'w-80'
  height?: string; // 예: 'h-40'
```
아예 tailwind를 활용하는 것은 어떠신가요?

Reply:
저 부분에 tailwind를 직접 넣기는 어렵고
```
width?: sm | md | lg;
height?: sm | md | lg;
```
와 같은 형태로 작성하여 각 키워드에 맞는 스타일 객체를 작성할 수 있을 것 같습니다.

그러려면 디자인 단계에서 해당 컴포넌트의 크기 지정을 어떤 식으로 할 건지 맞춰볼 필요가 있습니다. @ymj307